### PR TITLE
fix(doc): simpler doc on how to switch themes

### PR DESCRIPTION
**Pull request by nojhan**
+ remove deprecated/confusing files

Refs: #765

Not sure how to handle the doc linter complaining about `lp` in a code block being a spell check error?

### CODE REVIEW

File: docs/theme.rst [hunk 1]
```diff
@@ -28,7 +28,21 @@ If you want to create your own theme, see :doc:`theme/custom`.
...
 
 Switching Themes
-----------------
+================
+
+In a nutshell
+-------------
+
+To use a theme, just source its file in your shell configuration file,
+after having load the liquidprompt, and then call `lp_theme`:
```
Comment (Rycieos) [inline docs/theme.rst:37]: This should be double backtics:
And, to make the following a code block (and therefore ignore spelling issues), you need two colons:
```suggestion
after having load the liquidprompt, and then call ``lp_theme``::
```

**Review by Rycieos — CHANGES_REQUESTED** [review]
Looks good, just the fix and a rebase to `master` to fix Shellcheck issues.

File: liquidpromptrc-dist [hunk 2]
```diff
@@ -1,240 +0,0 @@
-
```
Comment (Rycieos) [inline liquidpromptrc-dist:1]: You are already deleting this file in #762, let's not duplicate work.

File: liquid.ps1 [hunk 3]
```diff
@@ -1,29 +0,0 @@
-
```
Comment (Rycieos) [inline liquid.ps1:1]: This file has a few references, they will need to be cleaned up:
```bash
± grep -RniI liquid.ps1
liquidprompt:4210:        # shellcheck source=liquid.ps1
docs/theme/default.rst:70:An example template file is available: liquid.ps1_.
docs/theme/default.rst:72:.. _liquid.ps1: https://github.com/nojhan/liquidprompt/blob/master/liquid.ps1
```

**Review by Rycieos — CHANGES_REQUESTED** [review]

File: liquid.ps1 (hunk 3 above)
Comment (nojhan) [reply liquid.ps1:1]: I added themes/minimal/minimal.ps1, which should be less confusing. It is just default minus a couple of sections.

**Comment by nojhan** [comment]
Rebased on master.

File: liquidprompt [hunk 4]
```diff
@@ -4207,7 +4275,7 @@ _lp_default_theme_prompt_data() {
 
 _lp_default_theme_prompt_template() {
     if [[ -f "${LP_PS1_FILE-}" ]]; then
-        # shellcheck source=liquid.ps1
+        # shellcheck source=themes/minimal/minimal.ps1
```
Comment (Rycieos) [inline liquidprompt:4278]: >         source "$LP_PS1_FILE"
>                ^------------^ SC1091 (info): Not following: themes/minimal/minimal.ps1: openBinaryFile: does not exist (No such file or directory)

File: docs/theme/default.rst [hunk 5]
```diff
@@ -67,9 +67,9 @@ For a template file to be loaded, its filepath must be set in
 A template file does nothing more than set :attr:`LP_PS1` to a value. The
 following sections are available to be used.
 
-An example template file is available: liquid.ps1_.
+An example template file is available: minimal.ps1_.
 
-.. _liquid.ps1: https://github.com/nojhan/liquidprompt/blob/master/liquid.ps1
+.. _minimal.ps1: https://github.com/nojhan/liquidprompt/blob/master/themes/minimal/minimal.ps1
```
Comment (Rycieos) [inline docs/theme/default.rst:72]: This file does not exist.

File: tools/theme-preview.sh [hunk 6]
```diff
@@ -126,6 +126,7 @@ _lp_hostname() {
 }
 _lp_connection() {
   lp_connection=ssh
+  SSH_CONNECTION="192.168.0.254 56314 192.168.0.1 22"
```
Comment (Rycieos) [inline tools/theme-preview.sh:129]: This change was already added in 8ea315e. Either drop the last commit, or rebase to latest `master`.

**Review by Rycieos — CHANGES_REQUESTED** [review]

File: liquidprompt (hunk 4 above)
Comment (nojhan) [reply liquidprompt:4278]: Sorry :man_facepalming:

File: themes/minimal/minimal.ps1 [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+PS1="${LP_PS1_PREFIX}"
+# add user, host, permissions colon, working directory, and dirstack
+PS1+="${LP_BRACKET_OPEN}${LP_USER}${LP_HOST}${LP_PERM}${LP_PWD}${LP_BRACKET_CLOSE}"
+
+# Add VCS infos
+# If root, the info has not been collected unless LP_ENABLE_VCS_ROOT
+# is set.
+PS1+="${LP_VCS}"
+
+# add return code and prompt mark
```
Comment (Rycieos) [inline themes/minimal/minimal.ps1:11]: Most of these comments are incorrect:
> add title escape time, jobs, load and battery

None of that is added.
> dirstack

That isn't in there either.
> add return code and prompt mark

Runtime is also added here.

**Review by Rycieos — CHANGES_REQUESTED** [review]

File: themes/minimal/minimal.ps1 (hunk 7 above)
Comment (nojhan) [reply themes/minimal/minimal.ps1]: Fixed the related comments in `liquidprompt` as well.

File: liquidprompt [hunk 8]
```diff
@@ -4293,7 +4293,7 @@ _lp_default_theme_prompt_template() {
         # is set.
         PS1+="${LP_VCS}"
 
-        # add return code and prompt mark
+        # Add last runtime, return code & meaning, prompt mar and user-defined postfix.
```
Comment (Rycieos) [inline liquidprompt:4296]:
```suggestion
        # Add last runtime, return code & meaning, prompt mark and user-defined postfix.
```

**Review by Rycieos — CHANGES_REQUESTED** [review]
Looks good, other than the typo.

**Comment by nojhan** [comment]
Rebased on current master.

**Pull request merged**